pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -498,25 +646,35 @@ func (r *hibernationReconciler) setHibernatingCondition(cd *hivev1.ClusterDeploy
 		controllerutils.ErrorScrub(errors.New(message)),
 		controllerutils.UpdateConditionIfReasonOrMessageChange,
 	)
-
-	if reason == hivev1.SyncSetsNotAppliedReason {
-		defer func() {
-			requeueAfter := timeBeforeClusterSyncCheck(cd)
-			logger.Infof("cluster will reconcile due to syncsets not applied in: %v", requeueAfter)
-			result.RequeueAfter = requeueAfter
-			result.Requeue = true
-			returnErr = errors.New(message)
-		}()
+	if changed {
+		logger.WithField("reason", reason).Info("Hibernating condition updated on cluster deployment")
 	}
+	return changed
+}
 
+func (r *hibernationReconciler) setReadyCondition(cd *hivev1.ClusterDeployment, reason, message string, status corev1.ConditionStatus, logger log.FieldLogger) bool {
```

Comment:
nit: This is so similar to the above, it doesn't really seem worth having a separate func.